### DISCUSSION TITLE
date-fns locale typescript problem

### DISCUSSION THREAD

**Discussion by ozanmanav** [gpbl/react-day-picker#1139] [Support]
Hi

I have a typing problem with date-fns.

I'm using parseDateForDatePicker for parsing date with format and locale with date-fns

![image](https://user-images.githubusercontent.com/19241756/63577133-69110480-c596-11e9-897d-e7416495be3e.png)

I'm when i check source code, i just seen locale is returning as a string but date-fns accepting Locale object, because of that i'm giving error like that;

![image](https://user-images.githubusercontent.com/19241756/63577252-ae353680-c596-11e9-9fef-013e448b0a34.png)

Thanks

**Comment by Racheal58** [comment]
Can you recreate your issue and attach a link  ??
